cpp/log/signer.cc

```diff
@@ -18,7 +19,9 @@ using cert_trans::Verifier;
 
 namespace cert_trans {
 
-Signer::Signer(EVP_PKEY* pkey) : pkey_(CHECK_NOTNULL(pkey)) {
+std::mutex lock_;
```

Comment:
This really needs to be a global singleton?!? In any case, it should be in an anonymous namespace, and the name changed to comply with the style guide (no trailing underscore, that's for member variables): https://google.github.io/styleguide/cppguide.html#Variable_Names

I'd change the name to be a bit more descriptive than just "lock", maybe "signer_lock"?